.travis.yml

```diff
@@ -28,7 +28,7 @@ script:
   - bundle exec rspec spec
   - bundle exec rubocop -V
   - bundle exec rubocop -F
-  - bundle exec brakeman -z -A
+  - bundle exec brakeman -c config/brakeman.ignore -z -A
```

Comment:
They do on your branch :) Here they make brakeman pass.

Brakeman fails because https://github.com/kubic-project/velum/pull/77/files#diff-b964d4ad50e36a0b0c7da1295058f697R112 is insecure. However, we already know that, so I had to create https://github.com/kubic-project/velum/pull/77/files#diff-1fb4db507da51ec0282450037ef38a47 in order for this warning to be omitted and have a green build.

I mean, we *know* insecure is insecure, that's why our flag is called `insecure` and is `false` by default. However, if someone wants to use an insecure SMT server (bad certificate or `http`) is their sole responsability.